### ISSUE TITLE
ImportError: cannot import name SiteProfileNotAvailable

### ISSUE THREAD

**Issue by olberg** [Liberationtech/django-libtech-emailuser#5]
Django 1.7 compatibility issue?

**Comment by oivvio** [comment]
Yes I noticed that. Unfortunately I will not get around to fixing that anytime soon... Or actually maybe I will cause I have it on a client project that I need to move to 1.7. But it will be at least a month or two. If you want to do it yourself I'll be happy to accept a pull request.

**Comment by pusnik** [comment]
Is possible that fix would be as simple to just delete get_profile function from models.py as its depracated in 1.7

**Comment by oivvio** [comment]
That is totally possible.

Try that out, make a PR and I'll accept it.

o

On Fri, Apr 10, 2015 at 3:00 PM, Grega Pusnik <notifications@github.com>
wrote:

> Is possible that fix would be as simple to just delete get_profile
> function from models.py as its depracated in 1.7
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Liberationtech/django-libtech-emailuser/issues/5#issuecomment-91547180>
> .
>



-- 
Oivvio Polite | +46 709 304030 | http://polite.se | skype:oivvio
